src/App.css

```diff
@@ -208,6 +207,9 @@ em {
 .side-text {
   margin-left: 2em;
 }
+.left-text {
+  max-width: 50rem;
+}
```

Comment:
This is maybe not a great name for the class (if I saw this class name, I'd assume it set `text-align:left;` or `float:left;`; you may want to name it something like `.constrainted-text`